### PR TITLE
Add browser theme to adwaita-maia-gtk3

### PR DESCRIPTION
**Pull request by Chrysostomus**
@oberon2007 I followed your suggestion to use symlink in bspwm-desktop-settings package

### CODE REVIEW

File: adwaita-maia-gtk3/PKGBUILD [hunk 1]
```diff
@@ -65,4 +67,10 @@ package() {
   ## Shift the hightlight-hue of all image files from blue to maia-greenish
   find . ! -type l -name "*.png" -exec mogrify -modulate 100,100,71 {} \;
   find . ! -type l -name "*.svg" -exec mogrify -background none -modulate 100,100,71 {} \;
+  
+  ## Install browser theme
+  mkdir -p $pkgdir/usr/share/themes/Firefox/Adwaita-Maia/
+  tar -zxvf ${srcdir}/chrome.tar.gz
+  cp -r ${srcdir}/chrome $pkgdir/usr/share/themes/Firefox/Adwaita-Maia/chrome/
+  rm -rf $pkgdir/usr/share/themes/Adwaita-Maia/chrome/
```
Comment (philmmanjaro) [inline adwaita-maia-gtk3/PKGBUILD]: This makes no sense at all.

**Comment by philmmanjaro** [comment]
Why a freaking tarball ?!? **chrome.tar.gz**

**Comment by oberon2007** [comment]
oops. sorry. accidentially just merged. reverted.

**Comment by Chrysostomus** [comment]
Ok, sorry for a bad commit, I'm learning the standards as I go. I made it into a tarball because I figured that it takes less space that way. Is it better to host it somewhere and make pkgbuild download it so it takes no space from this repo?

Can you give me more pointers about how to make it better? This version works, but seems not to be the correct way to do it.

**Comment by Chrysostomus** [comment]
And thanks for pointing this stuff out. Reason I sent pull request was because I was unsure about it.

**Comment by philmmanjaro** [comment]
Well, you can host those tarballs on your own github account. Create a project like **tarballs** and host them there. Only link to the raw-link then. This will keep our **.git** archive small. Since @oberon2007 merged it, that tarball is now anyway in our **.git** archive.

**Comment by Chrysostomus** [comment]
Okay. Again, sorry about this. Good thing it was only 170kb or so, but still, clutter is bad.

**Comment by oberon2007** [comment]
I think in this case it was not cathastrophical, since it was small. Anyway you can just park things like that in a saparate repo. Not even necessary to upload it as a tarball then, since github can do it for you when the PKGBUILD downloads it [like so](https://github.com/manjaro/packages-community/blob/master/artwork-andromeda/PKGBUILD#L13)

**Comment by philmmanjaro** [comment]
No it is fine. It is just to avoid binaries in git. That's all.

**Pull request merged**